src/data/parsing.py

```diff
@@ -272,63 +338,56 @@ def run(args_dict):
             if i%10==0:
                 print(i)
             allDocsClean.append(clean_doc(doc, spacynlp))
-            
+
         with open('allDocsClean.pkl', 'wb') as f:
             pickle.dump(allDocsClean, f)
 
 
-    allDocs2 = [eq.extract_and_replace_latex(doc) for docNum, doc in enumerate(allDocsClean)]
+    allDocs2 = [eq.extract_and_replace_latex(doc) for docNum, doc in
+                enumerate(allDocsClean)]
     print('Number of LateX Equations parsed: {}'.format(len(eq.LATEXMAP)))
-    
+
 
     ## Put equations back into text - this will be fed to glove embedding
-    if args_dict['createnewjson'].lower() == 'y' :
+    if args_dict['nlp_newjson']:
         print('***************\nBuilding JSON file for glove embedding...')
         allDocs3 = []
         percentCompletedMultiple = int(len(allDocs2)/10)
         for i, doc in enumerate(allDocs2[0:]):
-            if i%percentCompletedMultiple == 0: 
+            if i%percentCompletedMultiple == 0:
                 print('{}% completed'.format(round(i/(len(allDocs2))*100, 0)))
-            newDoc = reg.sub(r'Ltxqtn[a-z]{8}', eq.put_equation_tokens_in_text, doc)
+            newDoc = reg.sub(r'Ltxqtn[a-z]{8}', eq.put_equation_tokens_in_text,
+                             doc)
             allDocs3.append(newDoc)
 
         jsonObj2 = copy.deepcopy(jsonObj)
-        allDocs3Counter = 0 
+        allDocs3Counter = 0
 
         for key, value in list(jsonObj2.items()):
             if value['text']:
                 jsonObj2[key]['text']=allDocs3[allDocs3Counter]
                 allDocs3Counter = allDocs3Counter+1
-                
+
         with open('articles-with-equations.json', 'w', encoding='utf8') as fp:

```

Comment:
is it okay to just dump this in the current working directory, or should we specify a path (e.g., settings.data_dir)?

Reply:
tell me where it should go and i'll fix it up.

Reply:
Fixed.